region/lowernorfair/east/Lower Norfair Fireflea Room.json

```diff
@@ -286,6 +265,30 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [2, 2],
+      "name": "Leave Shinecharged",
+      "requires": [
+        {"canShineCharge": {
+          "usedTiles": 20,
+          "gentleDownTiles": 2,
+          "openEnd": 1
+        }},
+        {"or": [
+          {"enemyKill": {
+            "enemies": [["Fune"]]
+          }},
+          {"obstaclesCleared": ["A"]}
+        ]},
+        "canShinechargeMovement"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 120
+        }
+      },
+      "note": "FIXME: There may be an X-Mode strat that could work here without killing the Fune (possibly with Ice)."
```

Comment:
I couldn't get an xmode shinespark in the spike pit.  The entry angle doesn't work for it, plus it is very risky towards causing a crash.